tests/integration/test_analysisd/test_eps/data/configuration_template/configuration_enabled.yaml

```diff
@@ -0,0 +1,49 @@
+- sections:
+    - section: global
+      elements:
+        - limits:
+            elements:
+              - eps:
+                  elements:
+                    - maximum:
+                        value: MAXIMUM
+                    - timeframe:
+                        value: TIMEFRAME
+
+    - section: remote
+      elements:
+        - connection:
+            value: secure
+        - port:
+            value: '1514'
+        - protocol:
+            value: TCP
```

Comment:
Is this the default configuration?

Reply:
Yes it is. From the `ossec.conf` file
```
  <remote>
    <connection>secure</connection>
    <port>1514</port>
    <protocol>tcp</protocol>
    <queue_size>131072</queue_size>
  </remote>
```

Reply:
So why set it? In a EPS analysisd we shouldn't see any configuration of other modules